### DISCUSSION TITLE
Forked frontend is only displaying auction data for certain nouns

### DISCUSSION THREAD

**Discussion by ATXDAOEngineering** [nounsDAO/nouns-monorepo#834] [Frontend]
Hey ya'll!

We've deployed a set of smart contracts to goerli and hooked them into the frontend.

Interestingly enough, when we are looking through all the auctions, only some of their auction data will load correctly, meanwhile others are stuck in a perpetual "loading" state. 

Anyone have any clues or idea on why only some are loading correctly? 
Sorry if information is sparse, I'm not quite sure what to share or what would be helpful.

Thank you!

Properly loaded
<img width="1060" alt="Screenshot 2024-01-31 at 10 34 21 AM" src="https://github.com/nounsDAO/nouns-monorepo/assets/138176747/2456203d-96b9-4691-94e5-e3f66cc6e292">

Improperly loaded
<img width="1069" alt="Screenshot 2024-01-31 at 10 34 28 AM" src="https://github.com/nounsDAO/nouns-monorepo/assets/138176747/df0c6b14-7dbf-4a9c-b2aa-0f8a5de63b58">

Current Auction
<img width="1129" alt="Screenshot 2024-01-31 at 10 34 36 AM" src="https://github.com/nounsDAO/nouns-monorepo/assets/138176747/1cfd4210-5322-4dcd-b7c3-1fe0c3a40dd6">

**Comment by davidbrai** [comment]
I think there's a bug for past auctions with no bids.
hasn't happened in production for nouns, so wasn't a priority to fix it

**Comment by willyogo** [comment]
thanks for the quick answer @davidbrai! any other ideas for what could be causing this?

I can confirm that ATX Noun [6](https://goerli.etherscan.io/token/0x76d11501CD9EE0cc8384e1943027DE33864E323A?a=6) and [4](https://goerli.etherscan.io/token/0x76d11501CD9EE0cc8384e1943027DE33864E323A?a=4) do have bids and winners that aren't being displayed. 

Also just noticed something else that i have no doubt is related.. the frontend shows that ATX Noun 5, which in reality did not have any bids, was born on October 10, 2022, and the winner was 0xaD36c32F3c28A9214ADCf50A09998de1d3b0EE06. Sure enough, there is a [transaction from October 10, 2022](https://goerli.etherscan.io/tx/0x1c14a30919aac9406fa296b8261a74f2e331a941c4f8a9dc7b793b9d0158eee6) with a different deployment of NounsAuctionHouseProxy ([0xC143D0F48E0ba1e547Ff0fE489b4DC7A90063683](https://goerli.etherscan.io/address/0xc143d0f48e0ba1e547ff0fe489b4dc7a90063683)). I'm seeing the same issue with ATX Nouns 3 and 4, which did have real bids, so don't think this issue is related to whether or not an auction has bids.

It's strange, the current [auction](https://members.atxdao.com/auction) shows correct bid history, but there are two sets of issues I'm noticing with past auctions:

ATX Noun [1](https://goerli.etherscan.io/tx/0x494bd898e5c86dfc1dd15979877f3332dbe285e829da69ecda4a5b7a0cdda44d), [2](https://goerli.etherscan.io/tx/0xc889f97db42d41cb223841f44548371977ef9d515f42d0c167955973bac61249), and [6](https://goerli.etherscan.io/token/0x76d11501CD9EE0cc8384e1943027DE33864E323A?a=6) - Frontend displays no sale price, bid history, or owner. Frontend does display b.orn on October 9th or 10th 2022. ATX Noun 1 didn't have any bids, but 2 and 6 do, and were all born in the past few days.

ATX Noun 3, 4, and 5 - 3 and 4 have bids and winners, but 5 doesn't. For each of these, the frontend displays the bid history, owner, and sale price for the other deployment of NounsAuctionHouseProxy.

Also, the artwork displayed for each of these ATXNouns is incorrect and from the instance of NounsToken deployed back in October 2022. My gut says that the next thing we need to do is look at the difference between current auction data and past auction data, and make sure we're looking at the right contracts and/or subgraph. Thanks for reading 🙏  will keep digging into this and would really appreciate any guidance.

**Reply by davidbrai** [reply]
have you changed the subgraph url you're using?

**Reply by JacobHomanics** [reply]
We are using the default one provided from the initial fork.

Could this be the problem?

We should redeploy a subgraph of our own?

**Reply by willyogo** [reply]
deploying and pointing to our own subgraph sounds like the right next step to me

**Comment by willyogo** [comment]
I can see we still have this old instance of Nouns on Goerli in our config, which explains the above. Will update that with the correct goerli contracts and expect this will take care of it. Thanks again for the quick answer nonetheless @davidbrai!

https://github.com/ATXDAO/nouns-monorepo/blob/b2e2e64445ad4abdf65d121cc46abb831b44f444/packages/nouns-subgraph/config/goerli.json#L8
